.gitignore

```diff
@@ -61,6 +61,7 @@ test-results/
 !.yarn/plugins
 !.yarn/sdks
 !.yarn/versions
+**/.yarn/*
```

Comment:
Nit: We already do this on line 59 so is this necessary?

Reply:
`.yarn/*` only applies to  `.yarn` in root directory I had to add `**/.yarn/*` for subfolder with attribution generation script in `development/generate-attributions/.yarn`

Reply:
Ah! Right. Makes sense.